### PR TITLE
python: assert actual == expected ordering

### PR DESCRIPTION
**Pull request by erohmensing**
Encourage tests being written in the `assert expected == actual` order. Although [unittest is not opinionated about this](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual), PyCharm likes to show expected/actual diffs, so it is nice to define it in one direction. We picked the direction that Pycharm uses by default, though if we decided we wanted to go the other direction, this inference is overridable in Pycharm under `Advanced settings > Swap the order of actual and expected assertions in pytest`

### CODE REVIEW

**Comment by erohmensing** [comment]
* **#36980** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/36980?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @erohmensing and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_json_schema_helper.py [hunk 1]
```diff
@@ -230,7 +230,7 @@ def test_get_object_strucutre(object, pathes):
     ],
 )
 def test_get_expected_schema_structure(schema, pathes):
-    assert get_expected_schema_structure(schema) == pathes
+    assert pathes == get_expected_schema_structure(schema)
```
Comment (brianjlai) [inline airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_json_schema_helper.py:233]:
```suggestion
    assert paths == get_expected_schema_structure(schema)
```

nittiest of nits, but probably `paths`? technically unrelated to your change though

**Review by brianjlai — APPROVED** [review]
C O N S I S T E N C Y

File: airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_json_schema_helper.py (hunk 1 above)
Comment (erohmensing) [reply airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_json_schema_helper.py]: fixed!

**Pull request merged**